CONTRIBUTING.md

```diff
@@ -43,15 +43,25 @@ In that case, release a patched version to the [CHaP repository][CHaP], which al
 
 ## Troubleshooting
 
-If pretty much any test (launched with `cabal test`) fails with `<stdout>: commitBuffer: invalid argument (invalid character)` or `withFile: invalid argument (invalid byte sequence)`, make sure to execute `export LOCALE_ARCHIVE=/usr/lib/locale/locale-archive` in your shell (consider doing this automatically when entering the repository, with [direnv](https://direnv.net/)).
-
-## Releasing a version of the `cardano-cli`
+### `<stdout>: commitBuffer: invalid argument (invalid character)` error when launching tests
+If pretty much any test (launched with `cabal test`) fails with
+```
+<stdout>: commitBuffer: invalid argument (invalid character)
+```
+or
+```
+withFile: invalid argument (invalid byte sequence)
+```
+1. Make sure that you have file `/usr/lib/locale/locale-archive` present in your system.
+    * If you don't have `locale-archive` file present on your system, you need to generate it.
+      Please consult your system distrubution manual how to configure locale and generate that file.
```

Comment:
```suggestion
      Please consult your system distribution manual how to configure locale and generate that file.
```

Reply:
Forgot to enable spellcheckingin neovim. 🤦🏻 Thanks!